spec/Classes.tex

```diff
@@ -309,11 +309,14 @@ \subsection{Overriding Base Class Methods}
 dispatch in the event that a variable that has the base class type
 references an object that has the derived class type.
 
-The identical signature requires that the names, types, and order of
+The identical signature requires that the names, intents, types, and order of
 the formal arguments be identical. The return type of the overriding
 method must be the same as the return type of the base class's method,
 or must be a subclass of the base class method's return type.
 
+Methods that override a base class method must be marked with the
+\chpl{override} keyword in the \sntx{linkage-specifier}.
```

Comment:
Syntactically speaking, it feels weird to me to call `override` a linkage specifier since it has nothing to do with linking (`inline` arguably doesn't really either in retrospect...).  We could just rename `linkage-specifier` to `routine-kind` / `function-kind` or some other term that's more general.  Or we could have a general `routine-kind` nonterminal that maps down to `linkage-specifier` + `override` (+ `inline`?).

Maybe it's just that the one sentence goes by so quickly, but I wonder whether we should have a sentence or `\rationale` section saying a bit more about why this is the case (e.g., "This feature is designed to help avoid cases where class authors accidentally override a method without knowing it; or fail to override a method that they intended to due to not meeting the identical signature condition.").  

Or, another approach would be to add a sentence saying something like "Methods marked with `override` for which there is no parent class method with an identical signature will cause an error."  to capture the other side of the coin ("Must have it when overriding; can't have it when not overriding").  Though even then a rationale sentence might be valuable.